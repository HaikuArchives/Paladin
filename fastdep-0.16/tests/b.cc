#if 1-5+2*2+8-16/2
#include "b.h"
#endif

#if 10<20
#include "e.h"
#endif

#if 20==10
#include "f.h"
#include "notexistant.h"
#endif
