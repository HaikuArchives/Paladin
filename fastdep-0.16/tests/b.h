#ifndef B_H_
#define B_H_

#include "c.h"
#include "e.h"

#endif
