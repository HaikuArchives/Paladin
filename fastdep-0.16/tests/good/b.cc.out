b.o: b.cc \
	e.h
